cli/report/report.py

```diff
@@ -162,11 +195,17 @@ def file_jira_issues(
                 jira=firewatch_config.jira,  # type: ignore
             )
             issue_type = "Bug"
+
+            is_gitleaks_failure = False
+            if isinstance(pair["failure"], GitleaksDetectionsJobFailure):
+                is_gitleaks_failure = True
```

Comment:
```suggestion
            is_gitleaks_failure = isinstance(pair["failure"], GitleaksDetectionsJobFailure)
```

Reply:
I like this change and originally tried to use the `:=` operator here, but I receive this error when using Python 3.12.1.:

```
E       if self._gitleaks_detections := self._find_gitleaks_detections():
E          ^^^^^^^^^^^^^^^^^^^^^^^^^
E   SyntaxError: cannot use assignment expressions with attribute
```

It does not seem to like using `self` in this way, at least in the `__init__` method.